plugins/outputs/http/README.md

```diff
@@ -0,0 +1,34 @@
+# HTTP Output Plugin
+
+This plugin writes to a HTTP Server using the `POST Method`.
+
+Data collected from telegraf is sent in the Request Body.
+
+### Configuration:
+
+```toml
+# Send telegraf metrics to HTTP Server(s)
+[[outputs.http]]
+  ## It requires a url name.
+  ## Will be transmitted telegraf metrics to the HTTP Server using the below URL.
+  ## Note that not support the HTTPS.
+  url = "http://127.0.0.1:8080/metric"
+  ## http_headers option can add a custom header to the request.
+  ## The value is written as a delimiter(:).
+  ## Content-Type is required http header in http plugin.
+  ## so content-type of HTTP specification (plain/text, application/json, etc...) must be filled out.
+  http_headers = [ "Content-Type:application/json" ]
```

Comment:
I think we should do this in the same format as the httpjson input:
```toml
[outputs.http]
  [outputs.http.headers]
    Content-Type = "application/json"
```

This can also be written as an inline table, which is similar to the array version:
```toml
[outputs.http]
  headers = {Content-Type = "application/json"}
```

Reply:
I have changed to map[string]string.
awesome! :)